app/policies/tree_policy.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Policies for TreesController
+class TreePolicy < ApplicationPolicy
```

Comment:
I agree, too. I will delete the `TreePolicy` and tests and add a comment in `RepositoryPolicy`.